Vagrantfile

```diff
@@ -6,19 +6,19 @@ VAGRANTFILE_API_VERSION = "2"
 $script = <<EOF
 set -xe
 export DEBIAN_FRONTEND=noninteractive
+sudo apt-get update
 
-if [ ! -x "$(which erl)" -o $(($(date +%s) - $(stat -c '%Z' /var/lib/apt/periodic/update-success-stamp))) -gt 7200 ]; then
-	sudo sh -c 'echo "deb http://packages.erlang-solutions.com/debian wheezy contrib" > /etc/apt/sources.list.d/erlang.list'
-	wget -c http://packages.erlang-solutions.com/debian/erlang_solutions.asc
-	sudo apt-key add erlang_solutions.asc
-	rm -f erlang_solutions.asc
-	sudo apt-get update
-	rm -f erlang_solutions.asc
-fi
-
-sudo apt-get install -y --force-yes erlang git autoconf gcc-arm-none-eabi make build-essential
+sudo apt-get install -y --force-yes git autoconf gcc-arm-none-eabi make build-essential
```

Comment:
@proger why do we install `gcc-arm-none-eabi` here? Did you use it for building RPi images?